resources/assets/coffee/_classes/beatmap-discussion-helper.coffee

```diff
@@ -34,15 +38,13 @@ class @BeatmapDiscussionHelper
               currentBeatmapsetDiscussions.beatmapsetId == targetBeatmapsetDiscussions.beatmapsetId
             # same beatmapset, format: #123
             linkText = "##{targetBeatmapsetDiscussions.discussionId}"
-            attrs = 'class="js-beatmap-discussion--jump"'
+            options.classNames = ['js-beatmap-discussion--jump']
+            options.props.target = null
           else
             # different beatmapset, format: 1234#567
             linkText = "#{targetBeatmapsetDiscussions.beatmapsetId}##{targetBeatmapsetDiscussions.discussionId}"
 
-      linkText ?= displayUrl
-
-      "<a href='#{url}' rel='nofollow' #{attrs ? ''}>#{linkText ? url}</a>"
-
+      osu.link(url, linkText ? displayUrl, options)
```

Comment:
`osu.link` accepts unescaped data but strings here are escaped already resulting in double escaping for `&` and others.